website/src/index.css

```diff
@@ -434,6 +434,9 @@ nav.breadcrumbs li:last-of-type::after {
   header button {
     display: flex;
     justify-content: center;
+    position: absolute;
```

Comment:
this will position the element absolutely across all devices. I would prefer a grid-based solution, since that is less prone to breaking.

Reply:
@Idrinth I have implemented a grid-based solution. Please have a look.